src/components/EmojiPicker/EmojiPicker.js

```diff
@@ -0,0 +1,70 @@
+import React, { useState, useEffect } from 'react'
+import Picker from '@emoji-mart/react'
+import cx from 'classnames'
```

Comment:
i do want to keep alphabetizing imports, starting with third party libraries, then alphabetized imports from our codebase, so cx would go first, then emoji, then react